gradle-baseline-java-config/resources/idea/intellij-java-palantir-style.xml

```diff
@@ -54,6 +56,7 @@
             <CLASS name="Type" />
             <CLASS name="Key" />
             <CLASS name="Id" />
+            <CLASS name="Identifier" />
```

Comment:
This was recently added to error-prone 

https://github.com/google/error-prone/blame/c481b3f9c2da112db36ccfcbf64e755261a127ab/core/src/main/java/com/google/errorprone/bugpatterns/BadImport.java#L74